Writer/CsvWriter.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Pim\Bundle\DataGeneratorBundle\Writer;
+
+/**
+ * Write CSV files
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CsvWriter implements WriterInterface
```

Comment:
a good first way to prepare the re-using the PIM parts, at some point, I would really love use real PIM exports in the Generator